keyboards/gck/num3x3rp2040/keyboard.json

```diff
@@ -0,0 +1,142 @@
+        "cols": ["GP2", "GP3", "GP4"],
+        "rows": ["GP26", "GP27", "GP28"]
+    },
+
+    "url": "www.GCK.ru",
+    "usb": {
+        "device_version": "0.0.1",
+        "pid": "0x1313",
+        "vid": "0x1209",
+        "force_nkro":true 
+    },
+
+    "features": {
+        "command": false,
+        "console": false,   
+        "extrakey": true,
+        "mousekey": false,            
+        "nkro": true,
+        "indicators":true,      
+        "rgblight":true,
+        "rgb_matrix": false             
+    },
+    
+    "indicators":{
+        "on_state": 1,
+        "num_lock":"GP25"
+    }, 
+    
+    "ws2812": {
+        "driver": "vendor",
+        "pin": "GP14"
+    },  
+
+    "rgblight": {
+        "led_count": 64,
+        "animations": {
+            "static_light": true,
+            "breathing": true,
+            "rainbow_mood": true,
+            "rainbow_swirl": true,
+            "snake": true,
+            "knight": true,
+            "christmas": true,
+            "static_gradient": true,
+            "rgb_test": true,
+            "alternating": true,
+            "twinkle": true
+        },
+        "driver":"ws2812"
+    },
+
+    "rgb_matrix": {
+        "animations": {
+            "alphas_mods": true,
+            "gradient_up_down": true,
+            "gradient_left_right": true,
+            "breathing": true,
+            "band_sat": true,
+            "band_val": true,
+            "band_pinwheel_sat": true,
+            "band_pinwheel_val": true,
+            "band_spiral_sat": true,
+            "band_spiral_val": true,
+            "cycle_all": true,
+            "cycle_left_right": true,
+            "cycle_up_down": true,
+            "cycle_out_in": true,
+            "cycle_out_in_dual": true,
+            "rainbow_moving_chevron": true,
+            "cycle_pinwheel": true,
+            "cycle_spiral": true,
+            "dual_beacon": true,
+            "rainbow_beacon": true,
+            "rainbow_pinwheels": true,
+            "raindrops": true,
+            "jellybean_raindrops": true,
+            "hue_breathing": true,
+            "hue_pendulum": true,
+            "hue_wave": true,
+            "pixel_fractal": true,
+            "pixel_flow": true,
+            "pixel_rain": true,
+            "typing_heatmap": true,
+            "digital_rain": true,
+            "solid_reactive_simple": true,
+            "solid_reactive": true,
+            "solid_reactive_wide": true,
+            "solid_reactive_multiwide": true,
+            "solid_reactive_cross": true,
+            "solid_reactive_multicross": true,
+            "solid_reactive_nexus": true,
+            "solid_reactive_multinexus": true,
+            "splash": true,
+            "multisplash": true,
+            "solid_splash": true,
+            "solid_multisplash": true
+        },
+        "driver": "ws2812",
+        "led_count": 64,
+        "layout": [
+            { "flags": 4, "matrix": [0, 0], "x": 0, "y": 1 },
```

Comment:
Given you only have 9 keys here, im guessing the board doesnt actually have 64 LEDs?